components/style/properties/helpers/animated_properties.mako.rs

```diff
@@ -3012,6 +3020,19 @@ impl Animatable for IntermediateSVGPaintKind {
             _ => Err(())
         }
     }
+
+    #[inline]
+    fn get_zero_value(&self) -> Option<Self> {
+        match self {
+            &SVGPaintKind::Color(_) => {
+                Some(SVGPaintKind::Color(IntermediateRGBA::transparent()))
```

Comment:
Now that I look at this I suspect we could have made this a little simpler by doing:

```rust
    &SVGPaintKind::Color(ref color) => color.get_zero_value()
                                            .map(SVGPaintKind::Color)
```

or something like that. If you have a chance to try that, it might eliminate some redundant code.